web/resolvers/user_resolver.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Webapp.UserResolver do
```

Comment:
Is part of the GraphQL schema for queries. Take a look at web/schema.ex. The resolvers are called from in there.